crates/subspace-node/src/import_blocks_from_dsn.rs

```diff
@@ -1,267 +0,0 @@
-// Copyright (C) 2021 Subspace Labs, Inc.
```

Comment:
Was removing of this actually necessary? It would work about the same way if we implement it in the future, I see no immediate reason to remove it.

Reply:
We have it in the history and can reuse this code anyway, but since 'import from DSN' is an obsolete feature - IMHO it's better to remove it. Please, let me know whether you'd like to have it restored.

Reply:
Why is it obsolete? AFAIK this is roughly the way node will sync from DSN in the future.

Reply:
We discussed the issue offline: we will reuse this code in a similar fashion. I will revert the commit.